frappe/public/js/frappe/ui/toolbar/about.js

```diff
@@ -9,6 +9,12 @@ frappe.ui.misc.about = function() {
 			Website: <a href='https://frappe.io' target='_blank'>https://frappe.io</a></p>\
 		<p><i class='fa fa-github fa-fw'></i>\
 			Source: <a href='https://github.com/frappe' target='_blank'>https://github.com/frappe</a></p>\
+		<p><i class='fa fa-linkedin fa-fw'></i>\
```

Comment:
Most of the latest blogs, help articles, and release logs are shared via official social media. This will enable users to easily discover the official pages to follow on their favourite social media. Also, Adding official sources within the product will help users to avoid any fake pages.